wsclient/src/main/java/org/hpccsystems/ws/client/antlr/EclRecordReader.java

```diff
@@ -396,6 +396,68 @@ else if (currentrec != null)
         }
     }
 
+
+    @Override
+    public void enterSetdefaultvalall(EclRecordParser.SetdefaultvalallContext ctx) {
+        String val = ctx.getChild(2).getText();
+        val = val.replace("'", "");
+
+        if (currentfield != null)
+        {
+            currentfield.setColumnValue(val);
+        }
+        else if (currentrec != null)
+        {
+            currentrec.setColumnValue(val);
+        }
+    }
+
+
+    /**
+     * {@inheritDoc}
+     *
+     * <p>
+     * When entering a DEFAULT option for Set Of, set the columnvalue of the current field/rec to its value
+     * </p>
+     */
+    @Override
+    public void enterSetdefaultval(EclRecordParser.SetdefaultvalContext ctx)
+    {
+        String val = ctx.getChild(2).getText();
+        if(val.length() >= 2){
+            if(val.substring(0, 2).equals("['")) {
+                val = val.substring(2, val.length());
+            }
+        }
+        if(val.length() >= 2){
+            if(val.substring(val.length()-2, val.length()).equals("']")) {
+                val = val.substring(0, val.length() - 2);
+            }
+        }
+        //val = val.replaceFirst("(\[')", "");
```

Comment:
Minor: formatting differs here, and might want to remove these commented out lines.

Reply:
removed commented code